02Lesson/02Project/index.html

```diff
@@ -1,10 +1,50 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css"/>
+  <title>Daily Planet</title>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="css/style.css" type="text/css">
 </head>
 <body>
-  <!-- You code start here -->
+  <header>
+    <h1>
+      Daily Planet
+    </h1>
+    <nav>
+      <h2>

```

Comment:
Usually you would use `h2` tags only for real headlines, not to build a navigation bar. But since we were experimenting, this is fine here...